src/theme-default.ts

```diff
@@ -41,8 +41,12 @@ export default css`
     text-align: left;
   }
 
-  .content > ul > li {
-    margin-bottom: 1em;
+  ul > li {
+      margin-bottom: 0.5em;
+  }
+
+  ul ul > li:last-of-type {
+      margin-bottom: 0.4em;
```

Comment:
就统一成所有 li 都 0.5em 吧，简单明了易覆盖

```suggestion
  li {
      margin-bottom: 0.5em;
```